includes/classes/admin/mapping-wizard.php

```diff
@@ -156,8 +156,8 @@ public function add_help_tabs() {
 
 		$screen->set_help_sidebar(
 			'<p><strong>' . __( 'For more information:', 'content-workflow' ) . '</strong></p>' .
-			'<p><a href="https://gathercontent.com/support/wordpress-integration/" target="_blank">' . __( 'Content Workflow WordPress Integration' ) . '</a></p>' .
-			'<p><a href="https://wordpress.org/support/plugin/gathercontent-import" target="_blank">' . __( 'Support Forums' ) . '</a></p>'
+			'<p><a href="https://gathercontent.com/support/wordpress-integration/" target="_blank">' . __( 'Content Workflow WordPress Integration', 'content-workflow' ) . '</a></p>' .
```

Comment:
Sorry the ticket didn't explain but the "plugin slug" will be `content-workflow-by-bynder`

Reply:
Ah okay! I'll get that changed everywhere! Large number of file changes incoming!